x/nodes/keeper/abci.go

```diff
@@ -25,7 +26,7 @@ func BeginBlocker(ctx sdk.Ctx, req abci.RequestBeginBlock, k Keeper) {
 	// store whether or not they have actually signed it and slash/unstake any
 	// which have missed too many blocks in a row (downtime slashing)
 	signedBlocksWindow := k.SignedBlocksWindow(ctx)
-	minSignedPerWindow := k.MinSignedPerWindow(ctx)
+	minSignedPerWindow := k.MinBlocksSignedPerWindow(ctx)
```

Comment:
In `x/nodes/keeper/params.go`, we use `MinSignedPerWindow` and here we use `MinBlocksSignedPerWindo`.

Just confirming that this is intended?

Reply:
Yes. As described in [the Part2 commit](https://github.com/pokt-network/pocket-core/pull/1580/commits/9a11177eb35da369fd0e48566dd7a888303e8558)'s message, the function `k.MinSignedPerWindow` is renamed to `k.MinBlocksSignedPerWindow`.  This function returns `int64`, the number of blocks.

The parameter value `MinSignedPerWindow` (not a function), which is `BigDec` and the current value is `0.6`, is used in only one place, `GetParams` in x/nodes/keeper/params.go.